pycsw/wsgi.py

```diff
@@ -55,18 +30,53 @@
 import os
 import sys
 import six
+import re
+import ConfigParser
 from six.moves.urllib.parse import unquote
 
 from pycsw import server
 
 
 PYCSW_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+def dynUpdateConfigurationURLs(new_url):
```

Comment:
Please modify the code to follow Python's official style guide (AKA [pep8](https://www.python.org/dev/peps/pep-0008/)). Specifically:

* use *lower_case_separated_by_underscore* for naming functions and
    variables;
* leave two blank lines before defining a new function;
* do not write lines longer than 79 characters;
* use triple double quotes for function docstrings;